kube_apiserver_metrics/README.md

```diff
@@ -26,7 +26,7 @@ annotations:
     '[{ "prometheus_url": "https://%%host%%:%%port%%/metrics", "bearer_token_auth": "true" }]'
 ```
 
-Then the Datadog Cluster Agent schedules the check(s) for each endpoint onto Datadog Agent(s).
+Then the Datadog Cluster Agent schedules the check(s) for each endpoint onto Datadog Agent(s). You must add `cluster_check:true` to your [configuration file][9] when using a ConfigMap to configure cluster checks.
```

Comment:
I believe `cluster_check: true` is needed in static files as well as configmaps when running as cluster checks, so it may be better to move this addition down to line 32. Also, since the example of how to use it can be found in `https://docs.datadoghq.com/agent/cluster_agent/clusterchecks/#set-up-cluster-checks` , should we link to there instead?